e2e/api-spec.json

```diff
@@ -2076,12 +2115,12 @@
             ]
           }
         },
-        "x-schema-extension-multiple": {
-          "test": "test"
-        },
         "x-schema-extension": {
           "test": "test"
         },
+        "x-schema-extension-multiple": {
+          "test": "test*2"
+        },
```

Comment:
The order of these extensions changed but I modified the values to make it clear they aren't affected.